src/main/java/org/folio/service/orders/PurchaseOrderLineService.java

```diff
@@ -490,5 +499,26 @@ public Future<Void> validateAndNormalizeISBNCommon(List<CompositePoLine> composi
           .mapEmpty();
       });
   }
+
+  public Future<Void> updateSearchLocations(PoLine poLine, RequestContext requestContext) {
+    if (CollectionUtils.isEmpty(poLine.getLocations())) {
+      return Future.succeededFuture();
```

Comment:
if empty you should put empty array - case when all locations were removed

Reply:
done